resources/js/langs/en/index.js

```diff
@@ -1,17 +1,17 @@
-import common from './common.json'
-import creditcard from './creditcard.json'
-import litter from './litter.json'
-import location from './location.json'
-import notifications from './notifications.json'
-import profile from './profile.json'
-import { settings } from './settings/index'
-import { locations } from './locations/index'
+import common from './common.json';
+import creditcard from './creditcard.json';
+import litter from './litter.json';
+import location from './location.json';
+import notifications from './notifications.json';
+import profile from './profile.json';
+import { settings } from './settings/index';
+import { locations } from './locations/index';
 import { home } from './home/index';
-import signup from './signup.json'
-import tags from './tags.json'
-import teams from './teams.json'
-import nav from './nav.json'
-import upload from './upload.json'
+import signup from './signup.json';
+import tags from './tags.json';
+import teams from './teams.json';
+import nav from './nav.json';
+import upload from './upload.json';
```

Comment:
Please remove `;` they were left out intentionally

Reply:
Why you left them out? JS need ; at the end of the line.